modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManagerImpl.java

```diff
@@ -1577,6 +1586,8 @@ protected int fillQueueInternal(
         /** */
         private final GridSpinBusyLock busyLock;
 
+        private volatile AtomicBoolean nodeIsStopping;
```

Comment:
Hope that won't need such field at all in case of non-blockin partition reconciliation finalization or we can move it to partition reconciliation context.